docs/source/api/v1/phys_locations_id.rst

```diff
@@ -114,6 +117,12 @@ Response Structure
 			"state": "DC",
 			"zip": "20500"
 		}
+	],
```

Comment:
The Content-Length header needs to be updated.

Reply:
I can update it - but keep in mind I havent done this with any of my rewrites or deprecations (unsure if many ppl have) - I dont think it is a huge deal as its just an example response structure

Reply:
I asked around (aka docs overlord @ocket8888) and you're correct, I wouldn't bother updating.

Reply:
docs overlord thats hilarious! 🤣